tests/template_backends/test_dummy.py

```diff
@@ -55,6 +56,22 @@ def test_html_escaping(self):
         self.assertIn('&lt;script&gt;', content)
         self.assertNotIn('<script>', content)
 
+    def test_django_html_escaping(self):
+        if self.backend_name == 'dummy':
+            self.skipTest("test doesn't apply to dummy backend")
+
+        class TestForm(Form):
+            test_field = CharField()
+
+        media = Media(js=['my-script.js'])
+        form = TestForm()
+        template = self.engine.get_template('template_backends/django_escaping.html')
+        content = template.render({'media': media, 'test_form': form})
+
+        expected = '{}\n\n{}\n\n{}'.format(media, form, form['test_field'])
```

Comment:
But if in the future a `CharField` gets rendered differently or the default for Forms is now to display them as divs rather then as table for whatever reason, this test has to be changed as well.
Like this the test doesn't care what /exactly/ the html output is, just that it won't be escaped.